src/jetstream/repository/apikeys/psql_apikeys.go

```diff
@@ -111,6 +111,11 @@ func (p *PgsqlAPIKeysRepository) GetAPIKeyBySecret(keySecret string) (*interface
 		&apiKey.LastUsed,
 	)
 
+	// explicitly convert last_used to UTC
+	if apiKey.LastUsed != nil {
```

Comment:
RE this and lines 146-148... I think, given that we haven't released a version with this code in yet, it's ok to not update values already in the db. There's only a few of us that might have pre-existing values and it's straightforward to fix. The overhead of leaving these lines in is low, so if you prefer them in that's fine, just mark this comment as resolved

Reply:
Fair enough, less code is better I guess.